peers/test-utils/src/lib.rs

```diff
@@ -443,9 +444,9 @@ mod threaded_broadcaster_tests {
                 &local_signer_key,
             );
 
-            broadcaster.broadcast_consensus_msg(&msg1.issuer_responder_id(), &msg1);
-            broadcaster.broadcast_consensus_msg(&msg1.issuer_responder_id(), &msg1);
-            broadcaster.broadcast_consensus_msg(&msg1.issuer_responder_id(), &msg1);
+            broadcaster.broadcast_consensus_msg(&msg1, &msg1.issuer_responder_id());
```

Comment:
I flipped the argument order so that the `msg` is first. That felt more natural to me.